YoCode/Output.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace YoCode
+{
+    class Output
+    {
+        IPrint consoleWritter;
+
+        public Output()
+        {
+            consoleWritter = new PrintToConsole();
+            PrintIntroduction();
+        }
+
+        public void PrintIntroduction()
+        {
+            consoleWritter.AddNewLine(messages.Welcome);
+            consoleWritter.PrintMessage();
+        }
+
+
+        public void PrintFinalResults(List<FeatureEvidence> featureList)
+        {
+            foreach (var feature in featureList)
+            {
+                if (feature.EvidencePresent)
+                {
+                    consoleWritter.PrintDiv();
+                    consoleWritter.AddNewLine(feature.FeatureTitle);
+                    consoleWritter.AddNewLine("");
+                    consoleWritter.AddNewLine($"Feature implemented: {((feature.FeatureImplemented) ? "Yes" : "No")}");
+                    consoleWritter.AddNewLine("Indication of above: ");
+                    printEvidence(feature);
+                    consoleWritter.PrintDiv();
+                }
+                else
+                {
+                    consoleWritter.PrintDiv();
+                    consoleWritter.AddNewLine(feature.FeatureTitle);
+                    consoleWritter.AddNewLine($"Feature implemented: {((feature.FeatureImplemented) ? "Yes" : "No")}");
+                    consoleWritter.PrintDiv();
```

Comment:
Both branches start with consoleWritter.PrintDiv() and end with consoleWritter.PrintDiv(), you could do this before and after the branches so the branches themselves don't need to do it.

If you separate them to methods it would look something like:
```
consoleWritter.PrintDiv();
if (feature.EvidencePresent)
{
	produceOutputForEvidencePresent(feature);
}
 else 
{
    produceOutputForMissingEvidence(feature)
}
consoleWritter.PrintDiv();
```